examples/iot/iotoperations_broker/main.tf

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+terraform {
+  required_version = ">= 1.6"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.0"
+    }
+  }
+}
+provider "azurerm" {
+  features {}
+  subscription_id = "d4ccd08b-0809-446d-a8b7-7af8a90109cd"
```

Comment:
use a placeholder or variable, or just omit subscription_id field.

```suggestion
```

also applies to all other examples